test/csuite/random_directio/main.c

```diff
@@ -601,7 +627,7 @@ thread_run(void *arg)
 static void fill_db(uint32_t, uint32_t, const char *, uint32_t)
     WT_GCC_FUNC_DECL_ATTRIBUTE((noreturn));
 static void
-    fill_db(uint32_t nth, uint32_t datasize, const char *method, uint32_t flags)
+fill_db(uint32_t nth, uint32_t datasize, const char *method, uint32_t flags)
 {
 	WT_CONNECTION *conn;
 	WT_SESSION *session;
```

Comment:
Should there be `create` lines removed from here now? They're probably not harmful, but possibly confusing.